docs/Content/Files.md

```diff
@@ -1,56 +1,66 @@
 # Files
 
-When adding [items](Items.md) to your archive, often you will upload one or more files (images, documents, et al) associated with that item.
+When adding [items](Items.md) to your database, often you will upload one or more files (images, documents, etc.) associated with that item. An item can have as many files attached as you wish, or none. Omeka Classic does not offer a way to upload files that are not attached to an item, except for some branding files such as a header background or logo file, [depending on the theme configuration settings](../Admin/Appearance/Themes.md#configuring-a-theme).
 
-![Files tab of the Add an Item ](../doc_files/filesAdd.png)
+![The Files tab that appears when you are adding an item](../doc_files/filesAdd.png "The Files tab that appears when you are adding an item")
 
 File Types
 -------------------------------------------------------------
 
-Omeka Classic accepts most files and file types. If you are having difficulty or receiving MIME Type errors, please see more information about adjusting the MIME Types or Accepted File Extensions in the section on [managing security settings](../Admin/Settings/Security_Settings.md).
+Omeka Classic accepts most files and file types, and can be customized to accept or reject file types of your choice. You may wish to [format your multimedia files](#media-files) according to what can best be embedded and streamed in modern browsers.
+
+If you are having difficulty or are seeing file-validation errors, please see more information about adjusting the accepted file types and extensions in [File Validation section of the Security Settings page](../Admin/Settings/Security_Settings.md#file-validation).
+
+<!---
+Need more information about specific errors people have seen, or what "difficulty" might look like. Perhaps a screenshot or example here and more information in the Troubleshooting section.
+--->
+
 
 File Display Order
 ---------------------------------------------------------------
-Once files are added to an item, you may click and drag the files into the preferred display order for both public and admin item pages.
+If you have multiple files added to an item, you may click and drag the files into the preferred display order for both public and admin item pages.
+
+<!--- screenshot needed --->
 
 Files with Thumbnails
 -------------------------------
-Thumbnail previews are created for all files types in Omeka 2.0. If you have access to the config.ini file, you can manage [thumbnail configuration](../Technical/ConfiguringThumbnailCreation.md).
+Thumbnails are automatically created for all file types as of Omeka 2.0. If you have access to the `config.ini` file, you can manage [thumbnail configuration](../Technical/ConfiguringThumbnailCreation.md). 
+
+<!--- can you do anything fun with thumbnails? upload some defaults, upload some custom thumbnails to override the automatically generated ones? --->
```

Comment:
Want to go with "most" or do you have an example or two (or, ideally, a comprehensive list) of what's not supported?

Reply:
We make about half of the total number.

Reply:
@sharonmleon a threading issue again here I think, these comments are on the question about which files get thumbnails, "supported" in that sense

Reply:
@allanaaa the trick is that it's really just "whatever ImageMagick supports on your server" which can vary, and things like support for PDFs, video, rely on having both ImageMagick plus something else installed. If you're using GD that's a more knowable but more limited subset of things that get thumbnails. I might just say "Thumbnails are automatically created" and not elaborate beyond the link to the configuration page.

Reply:
Okay, here's how it stands:

> Thumbnails are automatically created for many file types as of Omeka 2.0. Thumbnail creation relies on the ability of your chosen thumbnail utility (the default being ImageMagick) and which file types it can process. If you have access to the `config.ini` file, you can manage [thumbnail configuration](../Technical/ConfiguringThumbnailCreation.md). Look up the utility you are using (such as [ImageMagick](https://imagemagick.org/){target=_blank}, [Imagick](https://www.php.net/imagick){target=_blank}, or [GD](https://www.php.net/manual/en/intro.image.php){target=_blank}) to find out which file types it supports. 

May be worth repeating on the "Configuring Thumbnails" page too.